pages/map.py

```diff
@@ -198,7 +259,7 @@ def create_user_emails_options(trips_group_by_user_id):
     Input('user-id-dropdown', 'value'),
 )
 def update_user_ids_options(trips_data, selected_user_ids):
-    user_ids_options, user_ids = create_user_ids_options(trips_data['users_data'])
+    user_ids_options, user_ids = create_user_ids_options(trips_data[0]['users_data'])
```

Comment:
why do we have a hardcoded `trips_data[0]` here?
Are we only plotting for the first user?